<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWidget</class>
 <widget class="QWidget" name="PlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlayerWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="VideoWidget" name="videoWidget" native="true"/>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="toolFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="Slider" name="positionSlider">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>12</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Slider" name="volumeSlider">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>12</height>
           </size>
          </property>
          <property name="value">
           <number>80</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="positionLabel">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>16</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(204, 204, 204);</string>
          </property>
          <property name="text">
           <string>00:00:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="durationLabel">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>16</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(111, 111, 111);</string>
          </property>
          <property name="text">
           <string>/00:00:00</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="searchComboBox">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>20</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>腾讯视频</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>search</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_isCachedPlay">
          <property name="minimumSize">
           <size>
            <width>68</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>68</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox#cb_isCachedPlay:!hover{
	background-color:rgba(255, 255, 255, 0); 
	color:rgb(170, 170, 170); 
}

QCheckBox#cb_isCachedPlay:hover{
	background-color:rgba(255, 255, 255, 0); 
	color:rgb(250, 250, 250); 
}</string>
          </property>
          <property name="text">
           <string>边下边播</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>slider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
